horizon_autotests/steps/volumes.py

```diff
@@ -384,7 +384,7 @@ def tab_backups(self):
 
     @pom.timeit('Step')
     def create_snapshot(self, volume_name, snapshot_name, description=None,
-                        check=True):
+                        check=True, modal_absent=True):
```

Comment:
Agree, ok let it be such variant now. But seems in future need to think about this parameter in `form.submit`. Actually I dislike, that need to forward variable, which doesn't have direct relationship to step.